### PR TITLE
Improve message styling

### PR DESCRIPTION
**Pull request by bradp**
One thing I dislike about the current submission success popup is that it's very intrusive. 

I think it's important to show these messages, both for confirmation that it was sent and also it makes you feel good providing data.

This PR moves the popup to the top of the screen. Makes it a bit bigger so it fits in nicely. Also changes to colors for named colors to actual hex codes to ensure consistency between browsers/OSes/etc. Actual color values have not been changed.

Happy to change the style / size of it.

Screen Shot 2022-10-03 at 10 28 43 AM

### CODE REVIEW

**Comment by DevJackSmith** [comment]
agree, but in the screenshot either the bar looks too big or the text is too small. Can we adjust that a little bit? Otherwise looks good :)

**Comment by DevJackSmith** [comment]
i used em instead of pixels where possible, since different people have different font size settings on different devices. Not sure if you still want to do your way, just fyi.

**Comment by bradp** [comment]
Yup - the only pixel values I’m using are for the box-shadow.

Let me bump the font size up a bit and see what ya think.

**Comment by DevJackSmith** [comment]
you are right, i was looking at your em, not mine haha.

**Comment by hymccord** [comment]
Could we also fix it not showing if two messages happen at once (ie hunt + convertible). Hunt message has priority and you don't get the confirmation that a convertible was submitted. Could need to be a separate PR though.

I like the style though!

**Comment by DevJackSmith** [comment]
> Could we also fix it not showing if two messages happen at once (ie hunt + convertible). Hunt message has priority and you don't get the confirmation that a convertible was submitted. Could need to be a separate PR though.
> 
> I like the style though!

good idea, i'll put it into another ticket

**Comment by DevJackSmith** [comment]
according to this browsers support color names: https://www.schemecolor.com/color-names-supported-on-all-web-browsers.php

File: src/scripts/main.js [hunk 1]
```diff
@@ -197,11 +197,9 @@
             mhhh_flash_message_div.css('background', 'lightgreen');
             mhhh_flash_message_div.css('border', '1px solid green');
         } else if (type === 'error') {
-            mhhh_flash_message_div.css('background', 'pink');
-            mhhh_flash_message_div.css('border', '1px solid red');
+            mhhh_flash_message_div.css('background', '#ffc0cb');
```
Comment (DevJackSmith) [inline src/scripts/main.js:200]:
```suggestion
            mhhh_flash_message_div.css('background', 'pink');
            mhhh_flash_message_div.css('border', '1px solid red');
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -200,8 +200,7 @@
             mhhh_flash_message_div.css('background', 'pink');
             mhhh_flash_message_div.css('border', '1px solid red');
         } else { // warning
-            mhhh_flash_message_div.css('background', 'gold');
-            mhhh_flash_message_div.css('border', '1px solid darkgoldenrod');
+            mhhh_flash_message_div.css('background', '#ffd700');
```
Comment (DevJackSmith) [inline src/scripts/main.js:203]:
```suggestion
            mhhh_flash_message_div.css('background', 'gold');
            mhhh_flash_message_div.css('border', '1px solid darkgoldenrod');```

File: src/scripts/main.js [hunk 3]
```diff
@@ -201,7 +201,7 @@
             mhhh_flash_message_div.css('border', '1px solid red');
         } else { // warning
             mhhh_flash_message_div.css('background', 'gold');
-            mhhh_flash_message_div.css('border', '1px solid darkgoldenrod');
+            mhhh_flash_message_div.css('border', '1px solid darkgoldenrod');```
```
Comment (DevJackSmith) [inline src/scripts/main.js:204]:
```suggestion
            mhhh_flash_message_div.css('border', '1px solid darkgoldenrod');
```

File: src/scripts/content.js [hunk 4]
```diff
@@ -17,11 +17,14 @@ mhhh_flash_message_div.setAttribute(
     "display:none;" +
     "z-index:100;" +
     "position:fixed;" +
-    "top:20%;" +
+    "top: 0;" +
     "background-color: white;" +
-    "padding: 10px;" +
-    "border-radius: 5px;" +
-    "box-shadow: 0 0 10px 1px black;");
+    "padding: 2em;" +
```
Comment (DevJackSmith) [inline src/scripts/content.js:22]:
```suggestion
    "padding: 1em;" +
    "font-size: larger;"
```

**Review by DevJackSmith — APPROVED** [review]
modified your pr a bit:
- removed hex for colors since browsers support color names
- adjusted font size to look a bit bigger with less padding

File: src/scripts/content.js [hunk 5]
```diff
@@ -17,11 +17,15 @@ mhhh_flash_message_div.setAttribute(
...
     "z-index:100;" +
     "position:fixed;" +
-    "top:20%;" +
+    "top: 0;" +
     "background-color: white;" +
-    "padding: 10px;" +
-    "border-radius: 5px;" +
-    "box-shadow: 0 0 10px 1px black;");
+    "padding: 1em;" +
+    "font-size: larger;"
```
Comment (DevJackSmith) [inline src/scripts/content.js:23]:
```suggestion
    "font-size: larger;" +
```

**Comment by DevJackSmith** [comment]
thank you!

**Pull request merged**